src/cephadm/cephadm

```diff
@@ -3590,6 +3583,10 @@ def _get_parser():
         '--skip-prepare-host',
         action='store_true',
         help='Do not prepare host')
+    parser_bootstrap.add_argument(
+        '--no-initial-service-specs',
+        action='store_true',
+        help='Do not create initial mon, mgr, and crash service specs')
```

Comment:
Hm. do we really want this? What about a `--no-deploy-crash` flag instead?

Reply:
Yeah, because qa/tasks/cephadm.py is carefully deploying specifically named daemons in specific places.

We probably want to teach it to have a less prescriptive mode, but for now that is the only way teuthology tasks work.

Reply:
Just for my understanding, the mgr daemon we created in bootstrap doesn't match the daemon we need according to teuthology and therefore we have to somehow work around it?

Reply:
It matches.  cephadm.py passes --mgr-id and we require that the teuth job have at least one mgr on the same host as the first mon.

We just don't want to have cephadm creating a second randomly-named mgr that doesn't match the teuth spec...